docker-action.yml.mustache

```diff
@@ -37,6 +37,13 @@ jobs:
 {{! Change delimiters to avoid the next line being parsed as mustache syntax. }}
 {{=<% %>=}}
           custom_image: ${{ matrix.image }}
+<%# ci_extra_dev %>
+          before_install: |
+            startGroup "Setup and print opam config"
+              opam repo -a --set-default add coq-extra-dev https://coq.inria.fr/opam/extra-dev
+              opam config list; opam repo list; opam list
```

Comment:
Is this listing of configuration and repos really necessary? By default, `before_install` does nothing, as far as I can see. In the other templates like for CircleCI, this listing is not done. I think we should try to be consistent across CI systems.

Reply:
> By default, `before_install` does nothing, as far as I can see.

It does. I just copied the default value here. See https://github.com/coq-community/docker-coq-action#before_install

Note that the way I extended the templates allowed me to have zero change to the docker-coq-action configuration at https://github.com/coq-community/goedel/pull/9/files (just added the `ci_extra_dev` variable to `meta.yml`). This code was originally written by you: https://github.com/coq-community/goedel/commit/867f0da7cc382d3e49606e4b080a4782fa1f14ab